Lib/test/test_abc.py

```diff
@@ -410,6 +410,34 @@ class C:
             with self.assertRaises(TypeError):
                 issubclass(C(), A)
 
+            # bpo-34441: Check that issubclass() doesn't crash on bogus classes
+            bogus_subclasses = [
+                None,
+                lambda x: None,
+                lambda: 42,
+                lambda: [42],
+            ]
+
+            for bs in bogus_subclasses:
```

Comment:
Nit: bs -> cls

Reply:
And it would be nice to use a `subTest()` here.

Reply:
> Nit: bs -> cls

Why? `bs` denotes (would-be) callables, not classes.

> And it would be nice to use a subTest() here.

Thank you, fixed.

Reply:
Then may be use `func`?

Reply:
OK!